### DISCUSSION TITLE
Viewing without parallax

### DISCUSSION THREAD

**Discussion by vinamragehlot** [isl-org/Open3D#3660] [Q&A] [answered]
Image 1 shows how Open3d visualizes by default with perspective view turned on, is there a way to show a true normal view such as show in image 2?

![1](https://user-images.githubusercontent.com/26214766/123688098-ef14cb00-d851-11eb-83cf-7daf09399967.jpg)
![2](https://user-images.githubusercontent.com/26214766/123688102-efad6180-d851-11eb-9661-4f7309c56e5e.jpg)

**Comment by germanros1987** [comment, marked as answer]
I guess now you can render using an Orthogonal camera: http://www.open3d.org/docs/0.12.0/python_api/open3d.visualization.rendering.Camera.html 

For instance:

```
render = o3d.visualization.rendering.OffscreenRenderer(img_width, img_height)
render.scene.camera.set_projection(o3d.visualization.rendering.Camera.Projection.Ortho, left, right, bottom, top, near, far)
```
